platform/features/table/src/controllers/TelemetryTableController.js

```diff
@@ -269,6 +263,11 @@ define(
                 }
 
             }
+
+            function getMetadataValues(metadata) {
```

Comment:
should be defined before use.

Reply:
My feeling here is that late definition of functions can improve readability of code. Defining a minor function like this before using it can be distracting to someone parsing the code - it's noise essentially. The details of how `getMetadataValues` works are less important than what is done with those metadata values in this section of code. Interested to hear your thought on this though.

Reply:
It's strange for me that I have to jump elsewhere to understand what the function is doing, and when I do find the function definition, it turns out it's a 1-liner.  Why not inline?

Reply:
Whether it's defined before or after the line it's used, there's still some jumping around involved, to me it makes more sense to have the function definition later, as it's less important. The name of the function should give a first-order description of what it does.

In terms of inlining vs having a separate function definition - In general I tend to favor separate functions just because of the noise involved in old style function definitions - but this is really just a personal preference. Happy to compromise here and inline it.